go/install/install_default.go

```diff
@@ -32,3 +32,8 @@ func StartUpdateIfNeeded(context.Context, logger.Logger) error {
 func GetNeedUpdate() (bool, error) {
 	return false, nil
 }
+
+// SnoozeUpdate will snooze the new update (if there is one) for 24 hrs.
+func SnoozeUpdate() error {
```

Comment:
for linux, it was discussed we might want to be able to snooze indefinitely

Reply:
I believe (jacob knows more about this) that this snoozing is done serverside. Should probably be handled clientside for linux. Could just set it in config.json maybe. Ideally there would be an undo button in the GUI as well but not a priority.

Reply:
Couple of things here.

* Snoozing is done on the server for `macOS` and `Windows` clients since they have `update` binaries bundled.

* Linux client have never snoozed historically because they don't have `installID`s. So calling directly to the API server here implicitly depends on an `installID`. We wouldn't want to do that.

* It's correct to call into the updater on `macOS` and `Windows` since it will handle/manage `installID`s.

* @heronhaye is correct that on Linux this 'snoozing' can only happen client side since we can't depend on the `install` or `update` binaries being present.

* `service/config.go` would want to perform a platform check (for linux), and instead of calling into `install` we would call something else that would write to `config.json` with the end snooze timestamp (epoch?). Subsequent checks for updates on Linux (handled in PICNIC-664) should read from `config.json` to see if we are before the snooze timestamp. Checking for an update after the snooze timestamp should clear the snooze setting automatically. This discussion thread can be carried into PICNIC-664.

* 'Undo' needs to go through design. Instead we can add a button to the GUI under settings that says 'Check for update'.

Reply:
Sounds fine. The install package already does the platform check so you can just put the config write here.

Reply:
great